tools/start-bindings/templates/build.gradle

```diff
@@ -8,3 +8,4 @@ android {
 ext.configureUniFFIBindgen("{{ crate_name }}")
 ext.dependsOnTheMegazord()
 ext.configurePublish()
+
```

Comment:
note this additional empty line is intentional, otherwise rinja will create a file with missing line break at the end of the file